### PR TITLE
Migrate | Migrates project to Yew

### PR DESCRIPTION
**Pull request by EstebanBorai**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "ruat-lang-ar-github-io"
```
Comment (igaray) [inline Cargo.toml:2]: typo

Comment (EstebanBorai) [reply Cargo.toml:2]: Thanks!

**Comment by EstebanBorai** [comment]
Update: The new design is being developed in this PR: https://github.com/rust-lang-ar/design/pull/1.
When the design is ready a styles sheet package will be created and will be used for this migration.

**Comment by EstebanBorai** [comment]
Closed in favor of #7

**Pull request closed without merging**